docs/ocis/getting-started/_index.md

```diff
@@ -71,6 +71,10 @@ docker run --rm -ti -p 9200:9200 -e OCIS_INSECURE=true owncloud/ocis
 When you're using oCIS with self-signed certificates, you need to set the environment variable `OCIS_INSECURE=true`, in order to make oCIS work.
 {{< /hint >}}
 
+{{< hint info >}}
+If you want to do `curl` commands to easily use/test the API manually, then you need to also set the environment variable `PROXY_ENABLE_BASIC_AUTH=true`.
```

Comment:
In theory, you also could use the OIDC client from the Rclone docs part with CURL (by setting the Bearer token in a header). In general we don't want to encourage to enable basic auth, because it bypasses the IDP. I think we should add a CURL chapter to the client section and list both possibilities (OIDC, Basic Auth) there.